xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1575,11 +1618,24 @@ Otherwise, the token value must be accessed from <xref:System.Text.Json.Utf8Json
         <Parameter Name="text" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="text">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="text">The text to compare against.</param>
+        <summary>Compares the text in a character span to the unescaped JSON token value in the source and returns a value that indicates whether they match.</summary>
+        <returns><see langword="true" /> if the JSON token value in the source matches the look up text; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If the look-up text is invalid or incomplete UTF-16 text (for example, it has unpaired surrogates), the method returns `false`,
```

Comment:
nit: since the remarks are the same for all three overloads, we could move this to a remarks inside a MemberGroup tag.